test/rubygems/test_gem_ext_builder.rb

```diff
@@ -228,20 +228,21 @@ def test_build_extensions_extconf_bad
                  @ui.output
     assert_equal '', @ui.error
 
-    gem_make_out = File.join @spec.extension_dir, 'gem_make.out'
+    make_out = File.read(File.join @spec.extension_dir, 'gem_make.out')
 
-    assert_match %r%#{Regexp.escape Gem.ruby}.* extconf\.rb%,
-                 File.read(gem_make_out)
-    assert_match %r%: No such file%,
-                 File.read(gem_make_out)
+    assert_match %r%#{Regexp.escape Gem.ruby}.* extconf\.rb%, make_out
+    assert_match %r%: No such file%, make_out
 
     refute_path_exists @spec.gem_build_complete_path
 
+    # windows Gem.ruby is SEPARATOR, gem_make.out is ALT_SEPARATOR
+    gem_ruby = !File::ALT_SEPARATOR ? Gem.ruby :
+      Gem.ruby.gsub(File::SEPARATOR, File::ALT_SEPARATOR)
+
     skip "Gem.ruby is not the name of the binary being run in the end" \
-      unless File.read(gem_make_out).include? "#{Gem.ruby}:"
+      unless make_out.include? "#{gem_ruby}:"
```

Comment:
Should we add parens here to continue the journey towards rubocop compiance?

Reply:
Re parens, I recall seeing it for method defs.  Is it also intended for method calls?

Reply:
@MSP-Greg Not sure if there's a way to differentiate them with Rubocop, so maybe? I was just thinking of https://github.com/rubygems/rubygems/pull/2492#issuecomment-441370126 where I added exceptions to several method calls to try to cut down the warnings.

Reply:
params are not required for method calls. And IMO it's not something that should be enforced with rubocop

Reply:
I'll change them, and thanks for the reminder to check it with RuboCop.  I often forget...

Reply:
@colby-swandale

Thanks for clarifying.  Sometimes I do, sometimes I don't...